tests/integration/molecule/module_check/converge.yml

```diff
@@ -197,3 +197,58 @@
     - assert:
         that:
           - result.objects == []
+
+    - name: Create check for idempotency test of complex fields
+      check:
+        name: complex
+        command: sleep 10
+        interval: 10
+        subscriptions:
+          - sub1
+          - sub2
+        handlers: []
+        runtime_assets: []
```

Comment:
Maybe we could omit one of these parameters in the first task and then set its value to `[]` in the idempotency check task, to test the reverse situation. This should work, right?

```
- name: Create check for idempotency test of complex field
  check:
    # handlers are null
    runtime_assets: []
```

and then

```
- name: Test for idempotency test of complex fields
   check:
     handlers: []
     # runtime_assets are null
```

Reply:
I thought I did this already. I guess not, Great catch.